QuestionAnswering/QA_SEQ2SEQ/deployment/fastapi_app/services/translate/data_processing/singapore/compact2verbose.py

```diff
@@ -0,0 +1,135 @@
+from services.translate.sparql import SparqlQuery
+from services.translate.sparql.graph_pattern import (
+    BindClause,
+    GraphPattern,
+    ServicePattern,
+    TriplePattern,
+)
+from services.translate.sparql.solution_modifier import GroupClause, SolutionModifier
+from services.translate.sparql.query_form import SelectClause
+from services.translate.sparql.where_clause import WhereClause
+
+
+class SgCompact2VerboseConverter:
+    def __init__(self, ontop_endpoint: str):
+        self.ontop_endpoint = ontop_endpoint
+
+    def _try_converty_haslanduse_triple(self, pattern: GraphPattern):
+        try:
+            """
+            ?Plot ozng:hasLandUseType/a ?LandUseTypeType .
+            """
+            assert isinstance(pattern, TriplePattern)
+            assert len(pattern.tails) == 1
+            subj = pattern.subj
+            pred, obj = pattern.tails[0]
+            assert pred == "ozng:hasLandUseType/a"
```

Comment:
See comment above. Several occurrences in this file.

Reply:
I've replaced the `assert` statement with `raise ValueError(...)`.